src/mobu/models/business.py

```diff
@@ -61,39 +51,72 @@ class BusinessConfig(BaseModel):
     settle_time: int = Field(
         10,
         title="How long to wait after lab creation in seconds",
-        description="Only used by the NotebookRunner",
+        description=(
+            "Wait this long after lag creation before trying to use the lab"
+        ),
         example=10,
     )
 
-    lab_idle_time: int = Field(
-        20,
-        title="How long to wait at end of lab loop in seconds",
-        description="Used by JupyterLoginLoop",
-        example=20,
+    idle_time: int = Field(
+        60,
+        title="How long to wait between business executions",
+        description=(
+            "AFter each loop executing monkey business, the monkey will"
+            " pause for this long in seconds"
+        ),
+        example=60,
+    )
+
+    login_idle_time: int = Field(
+        60,
+        title="Time to pause after spawning lab",
+        description=(
+            "Only used by JupyterLoginLoop and JupyterJitterLoginLoop."
+            " How long to wait after spawning the lab before destroying"
+            " it again."
+        ),
+        example=60,
     )
 
     execution_idle_time: int = Field(
-        0,
+        1,
         title="How long to wait between cell executions in seconds",
         description="Used by JupyterPythonLoop and NotebookRunner",
         example=1,
     )
 
     reauth_interval: int = Field(
-        2700,
+        30 * 60,
         title="Time between reauthentication attempts in seconds",
-        description="Used by JupyterLoginLoop, JupyterPythonLoop, and"
-        " NotebookRunner",
-        example=2700,
+        description=(
+            "Used by JupyterLoginLoop, JupyterPythonLoop, and NotebookRunner."
+            " JupyterHub appears to issue tokens with a one hour lifetime."
+        ),
+        example=30 * 60,
     )
 
     max_executions: int = Field(
         25,
-        title="How many cells to execute in a given kernel session",
-        description="Only used by JupyterPythonLoop",
+        title="How much to execute in a given lab and session",
+        description=(
+            "For JupyterPythonLoop, this is the number of code snippets to"
+            " execute before restarting the lab. For NotebookRunner, it's"
+            " the number of notebooks."
+        ),
```

Comment:
I suppose I could answer this myself, but does this turn into "number of notebooks times iterations-per-notebook" ?  Or is it "iterations of a notebook" ?

Reply:
I have now replaced this with:
```python
        description=(
            "For JupyterPythonLoop, this is the number of code snippets to"
            " execute before restarting the lab. For NotebookRunner, it's"
            " the number of complete notebooks. NotebookRunner goes through"
            " the directory of notebooks one-by-one, running the entirety"
            " of each one and starting again at the beginning of the list"
            " when it runs out, until it has executed a total of"
            " max_executions notebooks. It then closes the session (and"
            " optionally deletes and recreates the lab, controlled by"
            " delete_lab), and then picks up where it left off."
        ),
```